crates/primitives/src/constants.rs

```diff
@@ -14,3 +19,26 @@ pub const SEC_NONCE_SIZE: usize = 64;
 
 /// The size (in bytes) of a Hash (such as [`Txid`](bitcoin::Txid)).
 pub const HASH_SIZE: usize = 32;
+
+/// A verifiably unspendable public key, produced by hashing a fixed string to a curve group
+/// generator.
+///
+/// Note that this is _not_ necessarily a uniformly-sampled curve point!
+///
+/// But this is fine; we only need a generator with no efficiently-computable discrete logarithm
+/// relation against the standard generator.
```

Comment:
We should mention BIP 341 and link to it here.

Reply:
Fair enough. I removed the link since we aren't strictly using the offset technique, and the idea of hashing to a generator isn't unique to that BIP. But it could still be useful for a reader or reviewer, so I'll add it back.